kubernetes/gitlab.yaml

```diff
@@ -20,7 +20,7 @@ spec:
   selector:
     app: gitlab
     tier: frontend
-  type: NodePort
+  type: LoadBalancer
```

Comment:
does this work in minikube and bluemix free tier ?

Reply:
@paulczar you are right, lite(free) version only supports nodeport.

Reply:
I have not tested in minikube or Bluemix free tier, so I am not sure. I would assume that it will not work in minikube, because LoadBalancer uses the provider's load balancer to handle networking. I will test with Bluemix free tier.

In the meantime, using NodePort should have worked, except in my testing, it was not exposing the service.

Reply:
https://github.com/kubernetes/minikube/issues/384
^ Looks like LoadBalancer should work in minikube. I will still set some time aside to test it. ^

It is certainly true that not all providers offer LoadBalancer support.

Reply:
Bluemix free tier does not support LoadBalancer. However, since we specified nodeport under spec/port, the service should still expose to the nodeport with `type: LoadBalancer`.

Reply:
@mlangbehn You should able to use nodeport to access the gitlab ui. Make sure you are using the correct public ip from your cluster.

Reply:
@Tomcli: I am getting the IP from `bx cs workers [CLUSTER_NAME] |`. When I use NodePort, I can't reach the service. If I switch to LoadBalancer, I can use `kubectl get svc gitlab | grep gitlab | awk '{ print $3 }'` just fine.

I was also able to verify that using LoadBalancer does work with minikube, although it never actually gets an IP (just shows `<pending>` indefinitely) as mentioned in the linked minikube issue.

Reply:
Can we have a zoom session for this? Since it's your free account I can't access.

Reply:
Preferably in morning time PST :)